tracing/opentelemetry/src/main/java/io/helidon/tracing/opentelemetry/OpenTelemetrySpan.java

```diff
@@ -89,6 +96,33 @@ public Scope activate() {
         return new OpenTelemetryScope(delegate.makeCurrent());
     }
 
+    @Override
+    public Span baggage(String key, String value) {
+        Objects.requireNonNull(key, "Baggage Key cannot be null");
+        Objects.requireNonNull(value, "Baggage Value cannot be null");
+
+        Baggage.builder()
+                .put(key, value)
+                .build()
+                .storeInContext(getContext())
+                .makeCurrent();
+        return (io.helidon.tracing.Span) delegate;
```

Comment:
This is incorrect. You must return `this` (see all methods above). The delegate is not a helidon tracing span.

Reply:
@dalexandrov, it's NOT fixed! Please don't mark conversations as resolved if it's not fixed!